#!/usr/bin/env python
PACKAGE = "hektar"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("speed", int_t, 0, "wheel speed value", 100, 0, 127)
gen.add("variation_factor", double_t, 0, "scalar multiplier for wheel speed", 1.0, 0.0, 5.0)
gen.add("offset_multiplier", double_t, 0, "scalar multiplier for left wheel offset", 1.0, 0.0, 2.0)
gen.add("offset_addition", int_t, 0, "added value for left wheel offset", 0, -50, 50)

exit(gen.generate(PACKAGE, "hektar", "WheelControl"))

